docs/release/trg-0/kit-artifacts.md

```diff
@@ -0,0 +1,255 @@
+Business Partner KIT comes from the use case Business Partner Data Management.
+
+Status Quo / Todays challenge:  
+Most of today’s automotive companies have to invest a significant amount to keep their individual partner master data updated and correct, since outdated or incorrect data records result in federal fines and wrong claims / billing processes. The industry spends a vast amount into an area that is non differentiating, regulated and executed redundantly across the partners.
+
+For this, cutting individual company costs by offering the golden master data record as a shared service and proactively reduce Business Partner risk via collaborative value-add services.
+
+This will become the industry’s largest golden record collection as a shared and cost-optimized service (quality tested by the biggest players).
+
+OEMs and small-medium enterprises can:
+
+- Reduce today’s individual investments in master data mgt.
+- Re-allocate freed-up resources to differentiating / critical tasks
+- Time to value within 6 months, due to standard interfaces and mapping tools (e.g. SAP)
+- Increase data quality and use industry collective wisdom to act pro-actively (e.g. fraud alerts)
+
+Solution Provider:
+
+- Reduce investments to integrate due to one Data Model and open interface.
+- The KIT enables the offering of value-add services and innovation based on golden record and collective wisdom.
+- Potential to scale customer group and access new market potentials via marketplace and shared service network.
+
+### Tutorials
+
+Mandatory for the 'Graduated' stage. Optional for 'sandbox' and 'incubating'
+
+A tutorial provides educational resources that provides step-by-step guidance on how to use a KIT. Tutorials can be delivered in various formats, such as videos or written text. It may also provide additional resources for further exploration of the topic.
+
+Deliverables:
+
+- Description
+- Tutorial video or screenshot documentation
+
+### Whitepaper
+
+Optional
+
+A whitepaper outlines the overall objectives of a KIT regarding a specific business problem. It provides readers with background information on the topic, identifies key challenges and solutions to those challenges, evaluates alternative approaches to solving the business problem at hand, and offers recommendations for action.
+
+Deliverable:
+
+- Description
+- Link to the whitepaper
+
+### Semantic Models
+
+Mandatory for every KIT that is related to a use case + for the 'incubating' stage
+
+The semantic data models are a method of structuring data that includes semantic information that adds a basic meaning to the data and the relationships that lie between them to enable interoperability.
+
+Deliverables:
+
+- Description
+- Link to the semantic data models (GitHub)
+
+`Example from Traceability:`
+[Traceability Semantic Models](https://eclipse-tractusx.github.io/docs-kits/kits/Traceability%20Kit/Adoption%20View%20Traceability%20Kit#semantic-models)
+
+### Logic / Schema
```

Comment:
Logic and Schema of a technical application are a technical point, not a business point. Therefore, this shouldn't be located in the business view. I would vote for completely stroking this section.

Reply:
We will look into your points and optimize but for now I need to bring this stuff out.